### PR TITLE
#624 removed incorrect ALT clarification

### PR DESCRIPTION
**Pull request by d-cameron**

### CODE REVIEW

File: VCFv4.4.draft.tex [hunk 1]
```diff
@@ -322,9 +322,8 @@ \subsubsection{Fixed fields}
 
   \item ALT --- alternate base(s): Comma-separated list of alternate non-reference alleles.
   These alleles do not have to be called in any of the samples.
-  Options are base Strings made up of the bases A,C,G,T,N (case insensitive) or the `*' symbol (allele missing due to overlapping deletion) or a MISSING value `.' (no variant) or an angle-bracketed ID String (``$<$ID$>$'') or a breakend replacement string as described in Section \ref{Breakends}.
+  Options are base Strings made up of the bases A,C,G,T,N (case insensitive, non-zero length) or the `*' symbol (allele missing due to overlapping deletion) or a MISSING value `.' (no variant) or an angle-bracketed ID String (``$<$ID$>$'') or a breakend replacement string as described in Section \ref{Breakends}.
   If there are no alternative alleles, then the MISSING value must be used.
-  In other words, the ALT field must be a symbolic allele, or a breakend replacement string, or match the regular expression \texttt{\^{}([ACGTNacgtn]+|\string\*|\string\.)\$}.
```
Comment (tcezard) [inline VCFv4.4.draft.tex:327]: I'm guessing that this is incorrect because the sentence states that the ALT field follow this pattern where it is the alleles that does and the ALT field is a coma separated list of allele.
Wouldn't it better to change the `ALT` -> `allele` ? 

In any case this definition was also change in 4.3 and should be updated there as well.

File: VCFv4.4.draft.tex [hunk 2]
```diff
@@ -322,9 +322,8 @@ \subsubsection{Fixed fields}
 
   \item ALT --- alternate base(s): Comma-separated list of alternate non-reference alleles.
   These alleles do not have to be called in any of the samples.
-  Options are base Strings made up of the bases A,C,G,T,N (case insensitive) or the `*' symbol (allele missing due to overlapping deletion) or a MISSING value `.' (no variant) or an angle-bracketed ID String (``$<$ID$>$'') or a breakend replacement string as described in Section \ref{Breakends}.
+  Options are base Strings made up of the bases A,C,G,T,N (case insensitive, non-zero length) or the `*' symbol (allele missing due to overlapping deletion) or a MISSING value `.' (no variant) or an angle-bracketed ID String (``$<$ID$>$'') or a breakend replacement string as described in Section \ref{Breakends}.
```
[the remark this answers is not in the document]
Comment (jmarshall) [reply VCFv4.4.draft.tex:325]: (Most of hts-specs's documents use `nonzero` without a hyphen. But alternatively…)

“Options” here is a bit vague; with this sentence now being the only definition of ALT list items, I think the sentence would be clearer with a bit of rewording and more punctuation, e.g.

> Each allele in the list may be either: a non-empty string of bases (A,C,G,T,N case insensitively); the '*' symbol (allele missing due to overlapping deletion); a MISSING value '.' (no variant); an angle-bracketed ID String (''\<ID\>''); or a breakend replacement string, as described in Section \ref{Breakends}.
